### PR TITLE
mateu/add-stader-tag

### PR DESCRIPTION
**Pull request by mateumiralles**
- implementing stader tag option on mainnet and pratter networks

### CODE REVIEW

**Comment by mateumiralles** [comment]
/ValidatorsList:
image

/import:
image

File: packages/ui/src/ImportScreen.tsx [hunk 1]
```diff
@@ -165,6 +165,7 @@ export default function ImportScreen({
         { value: "rocketpool", label: "Rocketpool" },
```
Comment (pablomendezroyo) [inline packages/ui/src/ImportScreen.tsx:165]: Shouldnt be contemplated somewhere to dont allow to choose tag stader if network does not support stader?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: packages/ui/src/ImportScreen.tsx (hunk 1 above)
Comment (dappnodedev) [reply packages/ui/src/ImportScreen.tsx:165]: True, stader should only be available for prater and mainnet

Comment (dappnodedev) [reply packages/ui/src/ImportScreen.tsx:165]: Okay, it seems like this is already handled:
```
const tagSelectOptions: TagSelectOption[] = ["gnosis", "lukso"].includes(
    network
  )
    ? [{ value: "solo", label: "Solo" }]
    : ["holesky"].includes(network)
    ? [
        { value: "solo", label: "Solo" },
        { value: "rocketpool", label: "Rocketpool" },
        { value: "stakehouse", label: "StakeHouse" },
      ]
    : [
        { value: "solo", label: "Solo" },
        { value: "rocketpool", label: "Rocketpool" },
        
        { value: "stakehouse", label: "StakeHouse" },
        { value: "stakewise", label: "Stakewise" },
        { value: "stader", label: "Stader" },
      ];
```

Comment (pablomendezroyo) [reply packages/ui/src/ImportScreen.tsx:165]: LGTM

**Review by pablomendezroyo — APPROVED** [review]

**Review by dappnodedev — APPROVED** [review]

**Pull request merged**